pkg/config/config.go

```diff
@@ -19,20 +20,20 @@ type GuidPoolConfig struct {
 }
 
 func (dc *DaemonConfig) ReadConfig() error {
-	glog.Info("ReadConfig():")
+	log.Info().Msg("ReadConfig():")
 	err := env.Parse(dc)
 
 	return err
 }
 
 func (dc *DaemonConfig) ValidateConfig() error {
-	glog.Info("ValidateConfig():")
+	log.Debug().Msg("ValidateConfig():")
```

Comment:
perhaps print important config ? or the value of DaemonConfig with %+v ?

Reply:
I think in the read config you log the config with %+v